plugins/chat/assets/javascripts/discourse/components/chat/message-creator/new-group.gjs

```diff
@@ -26,7 +26,11 @@ export default class NewGroup extends Component {
       } else {
         return acc + 1;
       }
-    }, 1);
+    }, 0);
```

Comment:
This is important as we want to set the default count to 0/N, previously it was 1/N to account for current user.